docs/deployment/engine_share_level.md

```diff
@@ -42,15 +42,15 @@ With this feature, Kyuubi give you a more flexible way to handle different big d
 
 The current supported share levels are,
 
-|  Share Level   |            Syntax            |           Scenario           |               Isolation Degree               |      Sharability      |
+|  Share Level   |            Syntax            |           Scenario           |               Isolation Degree               |     Shareability      |
```

Comment:
According to Collins Dictionary. https://www.collinsdictionary.com/dictionary/english/shareability